lib/form/js/repeatable.js

```diff
@@ -4,257 +4,338 @@
 (function formRepeatable() {
   'use strict';
 
+  // Disabled one-var and func-names for better readability
```

Comment:
Yes. That is exactly why this rule exists.
http://eslint.org/docs/rules/func-names